src/main/java/io/molnarsandor/trelloclone/project/exceptions/ProjectNotFoundException.java

```diff
@@ -1,9 +1,9 @@
-package io.molnarsandor.pmtool.exceptions;
+package io.molnarsandor.trelloclone.project.exceptions;
 
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-@ResponseStatus(HttpStatus.BAD_REQUEST)
+@ResponseStatus(HttpStatus.NOT_FOUND)
 public class ProjectNotFoundException extends RuntimeException {
```

Comment:
Generikus `BusinessObjectNotFoundException` vagy saját (mert JPA-s kivétel már van ezen a néven) ˙EntityNotFoundException`-t javasolnék, amelyet bármilyen rekord meg nem léte esetén is tudsz dobni, persze paraméterezhetően, milyen entitás, milyen paraméterekkel rendelkező eleme nem található meg.